docs/source/sparse.rst

```diff
@@ -590,6 +590,7 @@ Torch functions specific to sparse Tensors
     sparse_csr_tensor
     sparse.sum
     sparse.addmm
+    sparse.sampled_addmm
```

Comment:
I think the docs build is complaining about the `_sparse.sparse_sampled_addmm` formatting? https://github.com/pytorch/pytorch/runs/5062506036?check_suite_focus=true

Reply:
Fixed now.
https://github.com/pytorch/pytorch/runs/5070979965?check_suite_focus=true